app/models/dor_harvester.rb

```diff
@@ -64,8 +66,10 @@ def on_error(resource, exception_or_message)
               else
                 exception_or_message.to_s
               end
-
-    RecordIndexStatusJob.perform_later(self, resource.bare_druid, ok: false, message: message)
+    RecordIndexStatusJob.perform_later(self,
+                                       resource.bare_druid,
+                                       ok: false,
+                                       message: truncate(message, limit: 1.megabyte))
```

Comment:
Actually, Rails makes `String#truncate` available, so we shouldn't need `TextHelper` at all.